org.gnome.NautilusPreviewer.json

```diff
@@ -0,0 +1,109 @@
+{
+    "app-id": "org.gnome.NautilusPreviewer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "sushi",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.a"
+    ],
+    "finish-args": [
+        "--socket=pulseaudio",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--talk-name=org.freedesktop.FileManager1",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
```

Comment:
Are these and DCONF_USER_CONFIG_DIR needed?

Reply:
No.

Reply:
Ok, removed